packages/zk-transfer-web/src/App.tsx

```diff
@@ -28,16 +29,24 @@ function App({ demo }: AppProp) {
     if (!state.chainId)
       demo.clientInit().then(async () => {
         console.groupCollapsed("Client Initialized");
-        const { zkTokens, thisAddressNotes } = await makeApi(demo);
+        const { zkTokens, thisAddressNotes, bm } = await makeApi(demo);
+
+        // Auto-Mint
+        console.info('Auto-Minting')
+        makeMint(demo, { bm, tradeSymbol: 'AAA', amount: 2 })
+        .then((payload) => { console.info("Minting Success:", payload)})
```

Comment:
To be consistent with style everywhere else, the `.then` and `.catch` should be indented two spaces.

Be sure to add an `eslint` script command in `package.json` and run it. See another package, like `packages/utils/package.json` for an example, and add a symlink to the `eslint.config.json` file in top-level `democracy` directory.